pkg/manager/member/tidb_member_manager.go

```diff
@@ -266,6 +266,31 @@ func getNewTiDBSetForTidbCluster(tc *v1alpha1.TidbCluster) *apps.StatefulSet {
 		})
 	}
 
+	sysctls := "sysctl -w"
+	if tc.Spec.TCPKeepaliveTime > 0 {
+		sysctls = sysctls + fmt.Sprintf(" net.ipv4.tcp_keepalive_time=%d net.ipv4.tcp_keepalive_intvl=%d", tc.Spec.TCPKeepaliveTime, tc.Spec.TCPKeepaliveIntvl)
+	}
+	if tc.Spec.SoMaxConn > 0 {
+		sysctls = sysctls + fmt.Sprintf(" net.core.somaxconn=%d", tc.Spec.SoMaxConn)
+	}
+
+	var initContainers []corev1.Container
+
+	if tc.Spec.TCPKeepaliveTime > 0 || tc.Spec.SoMaxConn > 0 {
+		privileged := true
+		initContainers = append(initContainers, corev1.Container{
+			Name:  "init",
+			Image: controller.GetSlowLogTailerImage(tc),
```

Comment:
Use another image or promote the slow log tailer image to a general util image

Reply:
Actually, the image does not matter, so I just reuse the image from the slow log tailer and we also have a default value for it, this does not involve any change to values.yaml and users do not need to know about it at all.